openapi/namsor/Dependencies.toml

```diff
@@ -0,0 +1,9 @@
+[[dependency]]
+org = "ballerina"
+name = "http"
+version = "1.1.0-beta.2"
+
+[[dependency]]
+org = "ballerina"
+name = "url"
+version = "1.1.0-beta.2"
```

Comment:
Both these versions have to be changed to `2.0.0`

Reply:
Updated